src/python/dxpy/asset_builder.py

```diff
@@ -79,22 +82,30 @@ def validate_conf(asset_conf):
     """
     if 'name' not in asset_conf:
         raise AssetBuilderException('The asset configuration does not contain the required field "name".')
-    # TODO: this default is not a good idea, and we will have to remove it once we ask customers to always provide release
-    if 'release' not in asset_conf:
-        asset_conf['release'] = "12.04"
-    elif asset_conf['release'] not in ['16.04', '14.04', '12.04']:
-        raise AssetBuilderException('The "release" field value should be either "12.04" (DEPRECATED), "14.04", "16.04".')
+
+    # Validate runSpec
+    if 'release' not in asset_conf or asset_conf['release'] not in ['16.04', '14.04', '12.04']:
+        raise AssetBuilderException('The "release" field value should be either "16.04", "14.04", or "12.04" (DEPRECATED)')
+    if 'runSpecVersion' in asset_conf:
+        if asset_conf['runSpecVersion'] not in ['0','1']:
```

Comment:
Nit: add space after comma and set " instead of ' for strings (here and in similar places in this commit). Thanks!